book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] To correctly complete the `many-to-many` (also `has-many-through`) relationship between `Person` and `Book` classes using the intermediate class `Rental`, kindly ensure that a method for adding rentals is present. It should look similar to this:
```ruby
class Book
   .
   .
   .
  def add_rental(person, date)
    # *the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. 
    Rental.new(date, self, person)
  end
end
```